architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -228,81 +350,146 @@ metadata:
   uid: ee2b9405-c68c-11e5-8a99-525400f25e34
 spec: {}
 status:
-  dockerImageRepository: 172.30.56.218:5000/test/origin-ruby-sample
+  dockerImageRepository: 172.30.56.218:5000/test/origin-ruby-sample <2>
   tags:
   - items:
-    - created: 2016-01-29T13:40:11Z
-      dockerImageReference: 172.30.56.218:5000/test/origin-ruby-sample@sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
+- created: 2017-09-02T10:15:09Z
+      dockerImageReference: 172.30.56.218:5000/test/origin-ruby-sample@sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d <3>
+      generation: 2
+      image: sha256:909de62d1f609a717ec433cc25ca5cf00941545c83a01fb31527771e1fab3fc5 <4>
+    - created: 2017-09-29T13:40:11Z
+      dockerImageReference: 172.30.56.218:5000/test/origin-ruby-sample@sha256:909de62d1f609a717ec433cc25ca5cf00941545c83a01fb31527771e1fab3fc5
       generation: 1
       image: sha256:47463d94eb5c049b2d23b03a9530bf944f8f967a0fe79147dd6b9135bf7dd13d
-    tag: latest
+    tag: latest <5>
 ----
-====
+
+<1> The name of the image stream.
+<2> The image source repository.
+<3> The internal registry service IP address, which is the target for pushes to the image stream. This value is set automatically.
+<4> The SHA identifier that this image stream tag currently references. Resources that reference this image stream tag use this identifier.
```

Comment:
And this one should be "The SHA identifier that this image stream tag previously referenced.  Can be used to rollback to an older image."